TU20Bot/Program.cs

```diff
@@ -24,17 +27,33 @@ internal class Program {
             server = new Server(client);
             handler = new Handler(client);
 
+            dbCommUnverifiedUser = new DbCommUnverifiedUser(new BotDbContext());
+            csvReader = new CSVReader(config);
+
             await handler.init();
-            
+
             await client.LoginAsync(TokenType.Bot, token);
             await client.StartAsync();
 
             // Run server on another thread.
             new Thread(() => server.RunAsync().GetAwaiter().GetResult()).Start();
 
+            // Run email checker on another thread every 15 min
+            new Thread(new ThreadStart(async () => {
+
+                do {
+                    // Reading and assigning data from csv file every 15 min
+                    config.userDataCsv = csvReader.readFile();
+
+                    await new EmailChecker(config, client, dbCommUnverifiedUser).emailCheck(config.userDataCsv);
+                    await Task.Delay(1800000);
+
+                } while (true);
+            })).Start();
```

Comment:
I cannot merge this with this here.

Please move this to any other event. If you need it to run every time the csv file changes, use [FileWatcher](https://docs.microsoft.com/en-us/dotnet/api/system.io.filesystemwatcher?view=netcore-3.1). If you need it to run every time the user runs a verify command, call it in the verify command.

This is too much.